crates/sncast/src/helpers/scarb_utils.rs

```diff
@@ -276,23 +259,40 @@ pub fn get_package_tool_sncast(metadata: &scarb_metadata::Metadata) -> Result<&V
     Ok(tool_sncast)
 }
 
+pub fn get_first_package_from_metadata(metadata: &Metadata) -> Result<PackageMetadata> {
```

Comment:
It terrifies me (I know this logic has been there before) xD Is it only used in tests? If so can we move it to the tests mod?

Reply:
It's used in tests in multiple modules